### PR TITLE
Added CourseRunState change endpoint.

### PR DESCRIPTION
**Pull request by waheedahmed**
[ECOM-7004](https://openedx.atlassian.net/browse/ECOM-7003)

Created endpoint to change course run workflow state.

### CODE REVIEW

**Comment by waheedahmed** [comment]
@tasawernawaz, @awais786 Please review.

**Review by tasawernawaz — APPROVED** [review]

**Comment by waheedahmed** [comment]
@bderusha Please review.

File: course_discovery/apps/publisher/api/urls.py [hunk 1]
```diff
@@ -2,12 +2,17 @@
...
+    CourseRoleAssignmentView, OrganizationGroupUserView, UpdateCourseKeyView, ChangeCourseRunStateView
 )
 
 urlpatterns = [
     url(r'^course_role_assignments/(?P<pk>\d+)/$', CourseRoleAssignmentView.as_view(), name='course_role_assignments'),
     url(r'^admins/organizations/(?P<pk>\d+)/users/$', OrganizationGroupUserView.as_view(),
         name='organization_group_users'),
     url(r'^course_runs/(?P<pk>\d+)/$', UpdateCourseKeyView.as_view(), name='update_course_key'),
+    url(
+        r'^course_runs/(?P<course_run_id>\d+)/change_state/(?P<pk>\d+)/$',
```
Comment (bderusha) [inline course_discovery/apps/publisher/api/urls.py]: Same question as the other PR.  It doesn't seem like we are using the `course_run_id` at all.  Would it make more sense to just expose these objects with urls like `r^course_run_state/(?P<pk>\d+)/$` and `r^course_state/(?P<pk>\d+)/$`

**Review by bderusha — APPROVED** [review]
👍 pending discussion of what the names of the endpoints should be

File: course_discovery/apps/publisher/api/urls.py (hunk 1 above)
Comment (waheedahmed) [reply course_discovery/apps/publisher/api/urls.py]: Done.

**Pull request merged**